### PR TITLE
Add blog post on ePages Hackathon

### PR DESCRIPTION
**Pull request by BirgitBader**
@RyanScovilleEpages please proof-read the blog post, thank you :-)

### CODE REVIEW

File: _posts/2016/2016-04-25-hackathon-2016.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+layout: post
+title: "ePages Hackathon 2016"
+date: "2016-04-25 07:16:17"
+icon: code
+categories: events
+authors: ["Birgit"]
+---
+
+Another year, another ePages hackathon.
+Spring is a nice season for hackdays, isn't it?
```
Comment (RyanScovilleEpages) [inline _posts/2016/2016-04-25-hackathon-2016.md]: hackdays -> hacking

hackdays is awkward

File: _posts/2016/2016-04-25-hackathon-2016.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+The world comes alive, the days are longer, and our developers have even more energy... for coding {% emoji wink %}!
+But what exactly did we do?
+
+## Ready, steady...
+
+{% image blog/blog-hack16-prep.jpg 50% right %} {% endimage %}
+
+This year's hackathon should focus on app development.
```
Comment (RyanScovilleEpages) [inline _posts/2016/2016-04-25-hackathon-2016.md]: should focus -> focused

keep tenses consistent

File: _posts/2016/2016-04-25-hackathon-2016.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+Now that this was clear, we started to collect more detailed topics and built individual teams.
+The Scrum Masters addressed all manner of things associated with the hackathon organisation:
+Meeting rooms had to be booked, working spaces to be coordinated, snacks and beverages to be organised, as well as the final topics had to be agreed.
```
Comment (RyanScovilleEpages) [inline _posts/2016/2016-04-25-hackathon-2016.md]: as well as the final topics had to be agreed. -> as well as agreeing upon the final topics.

File: _posts/2016/2016-04-25-hackathon-2016.md [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+
+And finally, it was time for the colleagues from Jena and Barcelona to travel to ePages headquarter in Hamburg.
```
Comment (RyanScovilleEpages) [inline _posts/2016/2016-04-25-hackathon-2016.md]: And finally, -> Finally,

Never start a sentence with a preposition

File: _posts/2016/2016-04-25-hackathon-2016.md [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+Prior to the event itself, some developers met in bars or clubs in Hamburg to outline their ideas, or prepare their GitHub repositories.
+
+## ...go!
+
+{% image blog/blog-hack16-agenda.jpg 50% left %} {% endimage %}
+
+At 9.00 a.m. sharp our CTO Harm Behrens kicked off the second ePages hackathon.
+Once the agenda was presented, the developers slit up into their teams, settled down and got hacking.
```
Comment (RyanScovilleEpages) [inline _posts/2016/2016-04-25-hackathon-2016.md]: slit -> split

settled down -> settled down,

File: _posts/2016/2016-04-25-hackathon-2016.md [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+The next 36 hours then basically consisted of three main activities: hacking, teamwork, and fun.
```
Comment (RyanScovilleEpages) [inline _posts/2016/2016-04-25-hackathon-2016.md]: basically -> REMOVE

File: _posts/2016/2016-04-25-hackathon-2016.md [hunk 7]
```diff
@@ -0,0 +1,49 @@
...
+
+Well, and also the challenge of feeding developers glued to their computers isn’t a small one.
```
Comment (RyanScovilleEpages) [inline _posts/2016/2016-04-25-hackathon-2016.md]: Well, and also -> REMOVE

too familiar, also awkward

File: _posts/2016/2016-04-25-hackathon-2016.md [hunk 8]
```diff
@@ -0,0 +1,49 @@
...
+Thankfully, tons of crisps, and cookies as well as pizza and lemonade helped to focus on the tough tasks they came to solve, while still having the energy to take breaks and clear their heads.
```
Comment (RyanScovilleEpages) [inline _posts/2016/2016-04-25-hackathon-2016.md]: crisps, and cookies -> crisps, cookies,

pizza and lemonade -> pizza, and lemonade

Always use the Oxford comma.  Always.
It reads like this:
crisps, cookies, pizza and lemonade.

As in, either crisps or cookies without lemonade, or pizza AND lemonade.  Don't do this.  :D

File: _posts/2016/2016-04-25-hackathon-2016.md [hunk 9]
```diff
@@ -0,0 +1,49 @@
...
+
+## Ship it!
+
+{% image blog/blog-hack16-result.jpg %} {% endimage %}
+
+After much weaving of code and pushing of pixels, and simply having a great time together, the teams presented their projects.
```
Comment (RyanScovilleEpages) [inline _posts/2016/2016-04-25-hackathon-2016.md]: , and simply having a great time together - > while having a great time together

File: _posts/2016/2016-04-25-hackathon-2016.md [hunk 10]
```diff
@@ -0,0 +1,49 @@
...
+The audience was highly impressed by those well-prepared presentations and all those fairly polished results.
```
Comment (RyanScovilleEpages) [inline _posts/2016/2016-04-25-hackathon-2016.md]: fairly -> remove

seems belittling
like the results weren't good enough

File: _posts/2016/2016-04-25-hackathon-2016.md [hunk 11]
```diff
@@ -0,0 +1,49 @@
...
+We saw native Android apps, developed in Java, cross-platform apps for Android and iOS, done with Ionic Framework or React Native, as well as Web apps.
```
Comment (RyanScovilleEpages) [inline _posts/2016/2016-04-25-hackathon-2016.md]: apps, developed -> apps developed

iOS, done -> iOS done

File: _posts/2016/2016-04-25-hackathon-2016.md [hunk 12]
```diff
@@ -0,0 +1,49 @@
...
+Next to this, some teams focused on other projects, like a REST cache layer or a complexity measuring tool or preparing ...
```
Comment (RyanScovilleEpages) [inline _posts/2016/2016-04-25-hackathon-2016.md]: Next to this, -> Meanwhile,

**Comment by BirgitBader** [comment]
superseeded by #488

**Pull request closed without merging**